providers/apis/binance/api_handler.go

```diff
@@ -20,14 +20,14 @@ var _ types.PriceAPIDataHandler = (*APIHandler)(nil)
 // https://github.com/binance/binance-spot-api-docs/blob/master/rest-api.md#public-api-endpoints
 type APIHandler struct {
 	// market is the config for the Binance API.
-	market mmtypes.MarketConfig
+	market types.ProviderMarketMap
```

Comment:
These comments could be a bit more specific.

"market is the market config for the Binance API"

"api is the api config for the Binance API"